pkg/remote/terraform/provider.go

```diff
@@ -125,6 +125,12 @@ func (p *TerraformProvider) configure(alias string) error {
 		"alias": alias,
 	}).Debug("New gRPC client started")
 
+	fmt.Printf("Terraform provider initialized (name=%s", p.Config.Name)
```

Comment:
Json output does not pipe the whole stdout, it will break if we use `--output json:///dev/stdout` but this is already broken actually. This is just a rework of the existing output by adding the name

Reply:
hmm ok so it was already broken. I think it's a little sad that we have an abstract output but allow to print on other places. But ok it's not this pr topic :+1:

Reply:
There is an open discussion about it there : https://github.com/cloudskiff/driftctl/discussions/56#discussioncomment-284917